mokagio/WordPress-iOS
# Consolidate all Danger workflows into one

See discussion here: https://github.com/wordpress-mobile/WordPress-iOS/pull/13476#issuecomment-587973780


To test this:

- Add or remove one or more label. If the PR ends up with no labels, Danger should post a comment about it
- Push a commit that adds a SHA reference in the `Podfile`, Danger should post a comment about it and fail the build

---

**No label + Code that Danger picks up as a failure:** two comments. _Note: I haven't found a way to consolidate the comments, because `danger ci` can only run one `Dangerfile` at a time._

<img width="1397" alt="Screen Shot 2020-02-22 at 13 25 00" src="https://user-images.githubusercontent.com/1218433/75084824-129d7680-5577-11ea-86ea-b0b727a1efa5.png">

**Only code that Danger picks up as failure:** only the comment about the failure.

<img width="1344" alt="Screen Shot 2020-02-22 at 13 26 42" src="https://user-images.githubusercontent.com/1218433/75084844-44164200-5577-11ea-8289-3c4a16b787e1.png">
